spec/support/rake.rb

```diff
@@ -1,16 +1,11 @@
 require 'rake'
 
 module RakeHelper
-  def load_rake_environment(files)
```

Comment:
The spec support for rake tasks was showing its age and complicating my ability to test *my* rake task (which calls other rake tasks) independent of testing the code in this helper. We now have simpler code, and less code, in this module.